default.env

```diff
@@ -1,8 +1,8 @@
 #######################################
 ## GLOBAL SETTINGS
 #######################################
-## enabe OLS UI
-OLS_ENABLE_UI=True
+## enable OLS UI
+ENABLE_DEV_UI=True
```

Comment:
thanks for bringing sanity/consistency here, but please keep the OLS prefix on the env var name.

Reply:
changed